src/js/views/SavedVideoView.js

```diff
@@ -0,0 +1,103 @@
+import { $ } from './utils.js';
+import { CONFIRM_MESSAGE, SELECTOR } from '../constants/index.js';
+import UserStorage from '../UserStorage/index.js';
+
+export default class SavedVideoView {
+  #savedVideoData;
+  #changeVideoWatchedStateBind;
+  #$playlist;
+  #$watched;
+
+  constructor() {
+    this.#$playlist = $(SELECTOR.PLAYLIST_VIDEO);
+    this.#$watched = $(SELECTOR.WATCHED_VIDEO);
+    this.#bindSavedVideoList();
+    this.appendVideos();
+  }
+
+  appendVideos() {
+    this.#savedVideoData = UserStorage.getVideoData();
+    this.#renderVideo();
+    this.#bindButtonEvent();
```

Comment:
```suggestion
$(SELECTOR.SAVED_VIDEO_LIST).addEventListener('click', this.#changeVideoWatchedStateBind);
```
굳이 메소드로 만들 필요가 있는지 궁금합니다. 외부에 노출되지도 않고 내부에서 재사용되지도 않고 
또한 타겟 + addEventListener + 콜백이 더 명시적이지 않나 생각됩니다.

Reply:
넵! 누락한 부분 수정과 함께 같이 수정했습니다!  [bb04ea9](https://github.com/woowacourse/javascript-youtube-classroom/pull/133/commits/bb04ea9d45bc43a1a1325c098bb9106067e5daae)